rules/windows/command_and_control_encrypted_channel_freesslcert.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/11/04"
+ecs_version = ["1.6.0"]
```

Comment:
No longer need to define this unless it is for anything other than the latest (which is currently 1.7.0)
```suggestion
```